themes/bootstrap3/js/lightbox.js

```diff
@@ -318,9 +318,9 @@ VuFind.register('lightbox', function Lightbox() {
       }, false);
     }
     // Loading
-    var lightboxHeader = $(_modalBody).find('.lightbox-header');
-    if (lightboxHeader.length > 0) {
-      lightboxHeader.after(VuFind.loading());
+    $(_modalBody).find('.modal-loading-overlay,.loading-spinner').remove();
+    if (_modalBody.children().length > 0) {
+      _modalBody.prepend('<div class="modal-loading-overlay">' + VuFind.loading() + '</div>');
     } else {
       _modalBody.prepend(VuFind.loading());
     }
```

Comment:
This isn't a fault with your improvement but I think we should move this to the ajax function so that links and forms both show the loading overlay. This is working beautifully for forms but not links thought this seems to be an existing issue.

Reply:
True, changed.